SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005902	Penalty is eased during combat incapability while buff is active.
+SKILL_20151224_005903	State of increased strength.
+SKILL_20151224_005904	State of continuos STA reduction.
+SKILL_20151224_005905	Hide
+SKILL_20151224_005906	Hidden state
+SKILL_20151224_005907	State of craft time reduction.
+SKILL_20151224_005908	Cyclone: Resist
+SKILL_20151224_005909	Increase harmful effect resistance rate
+SKILL_20151224_005910	Sacrifice HP recovery to 0 and raise magic amplification.
+SKILL_20151224_005911	Increase intelligence
+SKILL_20151224_005912	Increase defense
+SKILL_20151224_005913	Shield reduced instead of HP when hit by enemies
+SKILL_20151224_005914	Increase critical hit rate
+SKILL_20151224_005915	Increased critical hit rate
+SKILL_20151224_005916	Increase physical attack
+SKILL_20151224_005917	Increased physical attack
+SKILL_20151224_005918	Increase fire resistance
+SKILL_20151224_005919	Increased fire resistance
+SKILL_20151224_005920	Increase ice resistance
+SKILL_20151224_005921	Increased ice resistance
+SKILL_20151224_005922	Increase poison resistance
+SKILL_20151224_005923	Increased poison resistance
+SKILL_20151224_005924	Increase earth resistance
+SKILL_20151224_005925	Increased earth resistance
+SKILL_20151224_005926	Increase lightning resistance
+SKILL_20151224_005927	Increased lightning resistance
+SKILL_20151224_005928	Increase magical defense
+SKILL_20151224_005929	Increase magical attack
+SKILL_20151224_005930	Increased magical attack
+SKILL_20151224_005931	Porapore
+SKILL_20151224_005932	When stacked 100 times, increase holy and dark element by 6% of magic defense for 10 seconds.
+SKILL_20151224_005933	Porapore Activation
+SKILL_20151224_005934	Increase light and dark elemental attack
+SKILL_20151224_005935	Increase magical defense and decrease physical defense
+SKILL_20151224_005936	Spell Breaker
+SKILL_20151224_005937	HP won't be reduced by attacks until all the protections are used
+SKILL_20151224_005938	Shock Cutter
+SKILL_20151224_005939	Apply 5 hits to basic attack for every 5 [Bleeding].
+SKILL_20151224_005940	Shock Cutter: 5 Continuous Attack
+SKILL_20151224_005941	Changed to 5 continuous attacks
+SKILL_20151224_005942	Stamina reduction
+SKILL_20151224_005943	Cripple
+SKILL_20151224_005944	Token benefit: increase moving speed
+SKILL_20151224_005945	Moving speed increased
+SKILL_20151224_005946	Token
+SKILL_20151224_005947	 - Train 2 attribute skills at same time{nl} - Reduce market tax(30%{img white_right_arrow 16 16}10%){nl} - List maximum of 5 items on the market{nl} - Character moving speed +3{nl} - 1:1 trade enabled between token users{nl} - Purchasing shop enabled
+SKILL_20151224_005948	Nexon PC Bang
+SKILL_20151224_005949	Additional +30% Experience gained when defeating enemies{nl}Mark tax reduced by 20%{nl}Item listing count on market increased{nl}Train 3 attribute skills at same time{nl}Character Moving speed +3{nl}Stamina recovery doubled
```

Comment:
+30% Increase in EXP when defeating monsters{nl}Reduced Market sales tax by 20%{nl}Increased no. of item listings to the Market{nl}Learn up to 3 attribute skills simultaneously{nl}Movement speed increased by +3{nl}Doubled Stamina recovery rate